ontology/uco/observable/observable.ttl

```diff
@@ -4447,8 +4447,16 @@ observable:MobileDeviceFacet
 		[
 			sh:datatype xsd:integer ;
 			sh:maxCount "1"^^xsd:integer ;
+			sh:message "Deprecation notice: the object referenced by observable:storageCapacityInBytes will be required to be an observable:totalStorageCapacityInBytes in UCO 2.0.0."@en ;
 			sh:nodeKind sh:Literal ;
 			sh:path observable:storageCapacityInBytes ;
+			sh:severity sh:Warning ;
```

Comment:
*This* `sh:severity sh:Warning` needs to be removed.

Reply:
Also, the `sh:message` needs to be cut from this shape.

A `sh:message` is returned from SHACL validation when *any* constraint in the `sh:PropertyShape` is violated.  We don't want a custom message for this property shape; instead, we're just using "default" messages from whichever SHACL validator the user is using.